plugin.py

```diff
@@ -830,9 +830,9 @@ def karma(self, irc, msg, args, name):
 
         inc, dec = 0, 0
```

Comment:
do you need to also set ``alltime_inc, alltime_dec = 0, 0`` here and do away with the ``inc, dec = 0, 0`` line?